common/util/src/test/java/org/eclipse/dataspaceconnector/common/configuration/ConfigurationFunctionsTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.common.configuration;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junitpioneer.jupiter.ClearEnvironmentVariable;
+import org.junitpioneer.jupiter.ClearSystemProperty;
+import org.junitpioneer.jupiter.SetEnvironmentVariable;
+import org.junitpioneer.jupiter.SetSystemProperty;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ConfigurationFunctionsTest {
+
+    public static final String DEFAULT = "default";
+    public static final String VAR_1 = "VAR_1";
+    public static final String VAR_2 = "VAR_2";
+    public static final String VAR_3 = "VAR_3";
+    private static final String PROP_1 = "property1";
+    private static final String PROP_2 = "property2";
+    private static final String PROP_3 = "property3";
+    public static final String VALUE_1 = "value1";
```

Comment:
nit: Maybe we can keep the key name same e.g PROP_1 = "edc.key1"; and ENV_VAR_1 = "EDC_KEY1"; and so on. So that it goes with internal logic of  `propOrEnv` method to use the same key name only.

Reply:
good idea, changed